src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOpRequest.java

```diff
@@ -6,7 +6,7 @@
 import se.liu.ida.hefquin.engine.queryplan.logical.LogicalPlanVisitor;
 import se.liu.ida.hefquin.engine.queryplan.logical.NullaryLogicalOp;
 
-public class LogicalOpRequest<ReqType extends DataRetrievalRequest, MemberType extends FederationMember> implements NullaryLogicalOp
+public class LogicalOpRequest<ReqType extends DataRetrievalRequest, MemberType extends FederationMember > extends LogicalOperatorBase implements NullaryLogicalOp
```

Comment:
No need to add a space here.
```suggestion
public class LogicalOpRequest<ReqType extends DataRetrievalRequest, MemberType extends FederationMember> extends LogicalOperatorBase implements NullaryLogicalOp
```